src/components/EstimatedFee.tsx

```diff
@@ -49,9 +49,9 @@ export const EstimatedFee = (): React.ReactElement => {
   return (
     <div>
       <Typography variant="body1" color="secondary">
-        Estimated source Fee:
+        {`Estimated source Fee ${estimatedFeeLoading ? 'Loading' : ''}`}
```

Comment:
```suggestion
        {`Estimated ${chain} Fee: ${estimatedFeeLoading ? '...' : ''}`}
```
Can we replace this to say the chain name (uppercase), have a double colon so that it's clear that it's a label for some value?

I'd also get rid of `Loading`. It's not really "loading" anything, it's being calculated and we are waiting for a response. Adding a word there is extra mental capacity imho, I'd just replace it with dots. This is totally my preference though so I'm not super strong on this.

Reply:
I like using three dots instead of loading ....

Regarding the chain , what it would be better is to add the corresponding chain icon instead of the chain name.

Reply:
> Regarding the chain , what it would be better is to add the corresponding chain icon instead of the chain name.

I'm not saying to do that in this PR but as a refresher we have this issue: #48

Reply:
> I'm not saying to do that in this PR but as a refresher we have this issue: #48

I agree you should reopen that issue and lets do that there